js/samples/04.ai-apps/d.assistants-mathBot/sample.env

```diff
@@ -2,5 +2,6 @@
 BOT_ID=
 BOT_PASSWORD=
 
-OPENAI_KEY=
```

Comment:
same here, comment out

Reply:
good idea - i've added appropriate comments to `sample.env`, `.env.local.user` and `teamsapplocal.yml`.